app/models/in_progress_form.rb

```diff
@@ -9,11 +9,11 @@ def cast(value)
     alias serialize cast
   end
 
-  RETURN_URL_SQL = "CAST(metadata -> 'return_url' AS text)"
-  scope :has_attempted_submit, -> { where("(metadata -> 'submission' ->> 'has_attempted_submit')::boolean") }
+  RETURN_URL_SQL = "CAST(metadata -> 'returnUrl' AS text)"
```

Comment:
these scopes are only used in a rarely used rake task so I'm fine with not making them work with both cases.  They'll just work with new data.

Reply:
oh whoa! big oversight on my part. are you sure? we could something fancy:
```
where (return_url: x, updated_at: before_pr) OR where (returnUrl: x, updated_at: after_pr)
```
?

Reply:
I honestly don't think it's worth the effort.  Most of the time people are looking for recent issues with these scopes.